octoprint_slic3r/__init__.py

```diff
@@ -301,7 +301,8 @@ def do_slice(self, model_path, printer_profile, machinecode_path=None, profile_p
 				async_kwarg = 'async'
 			p = sarge.run(command, cwd=working_dir, stdout=sarge.Capture(), stderr=sarge.Capture(), **{async_kwarg: True})
 			p.wait_events()
-			try:
+                        last_error=""
+                        try:
```

Comment:
These lines were previously with tabs and now they are with spaces.  python cares very much about whitespace, I'm surprised that this didn't cause an error!

Let's be consistent and maintain the current usage of tabs everywhere, thanks!

Reply:
sry, my IDE is set for whitespace in python. fixed